app/javascript/containers/RestaurantContainer.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import RestaurantInfoTile from "../tiles/RestaurantInfoTile";
+
+class RestaurantContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      restaurant: {}
+    };
+  }
+
+  componentDidMount() {
+    let id = this.props.params.id;
+    fetch(`/api/v1/restaurants/${id}`)
+      .then(response => {
+        if (response.ok) {
+          return response;
+        } else {
```

Comment:
Please add an `else` action here (e.g., throwing to a `catch` you add after the final `then`